letters-to-symbols/letters_to_symbols.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+"""letters_to_symbols.ipynb
+
+Automatically generated by Colab.
+
+Original file is located at
+    https://colab.research.google.com/drive/1sTmLR7aUp2iMxCUH-R4wquV3L9Fv2We9
+"""
+
+def letters_to_symbols(s):
+  ret_str = ''
+  count = 1
+  pre_char = s[0]
+  for char in s[1:]:
+    if pre_char != char:
+      ret_str += str(count) + pre_char
+      count = 1
+      pre_char = char
+    else:
+      count += 1
+  ret_str += str(count) + s[-1]
```

Comment:
This solution will not work if the string ends in a single character that does not repeat (e.g. try using "AAAABBBCCDA")